oak-run/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/LuceneIndexer.java

```diff
@@ -100,9 +132,29 @@ public Set<String> getRelativeIndexedNodeNames() {
 
     @Override
     public void close() throws IOException {
+        LOG.info("Statistics: {}", formatStats());
+        binaryTextExtractor.logStats();
         indexWriter.close(System.currentTimeMillis());
     }
 
+    public String formatStats() {
+        long endTimeNanos = System.nanoTime();
+        long totalTimeNanos = endTimeNanos - startTimeNanos;
+        long nodesIndexed = progressReporter.getTotalUpdatesCount();
+        long avgTimePerDocumentMicros = nodesIndexed == 0 ? -1 : (totalTimeIndexingNanos / nodesIndexed)/1000;
+        long otherTimeNanos = totalTimeIndexingNanos - totalTimeMakingDocumentsNanos - totalTmeWritingToIndexNanos;
```

Comment:
Just a small thing but what is `otherTimeNanos`?

Reply:
Removed this metric, as most of the time it will be 0. And it can be derived from the others 
`totalTimeIndexingNanos = totalTimeIndexingNanos + totalTimeMakingDocumentsNanos + otherTime`